decide/voting/models.py

```diff
@@ -47,6 +82,18 @@ class QuestionOption(models.Model):
     hidden = models.BooleanField(default=False)
 
     def save(self):
+        if self.question.question_type == "B" and self.question.options.count() > 1:
+            if self.question.voteBlank:
+                if (
+                    self.option != "Sí"
+                    and self.option != "No"
+                    and self.option != "Voto En Blanco"
+                ):
```

Comment:
This does not make much sense, if you are creating a boolean voting but are still asking the user to input question options, how is it different from a "Single" voting? It should rather automatically, somehow create automatic question options and ignore whatever is inside, or if you manage to do so, which atm I can't recall how to do that, show only the question options fields in the admin page only if the voting is not boolean

Reply:
In case you have tested the functionality you will have noticed that the special case where the boolean vote is combined with the blank vote, must be handled differently but the automatic generation of questions prevails.

Reply:
I know it's not the best way to do it, but that's also out of my reach and time. Although if you add any option it raises a BadRequest error preventing you to do so.